Frontend/README.md

```
@@ -23,4 +23,36 @@
 
 
 ### Bugs
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)